pkg/image/registryclient/dockercredentials/credentials_test.go

```diff
@@ -0,0 +1,84 @@
+package dockercredentials
+
+import (
+	"net/url"
+	"reflect"
+	"testing"
+
+	"k8s.io/kubernetes/pkg/credentialprovider"
+
+	_ "github.com/openshift/origin/pkg/api/install"
+)
+
+type mockKeyring struct {
```

Comment:
Same as before, please remove these duplicates from `pkg/image/importer/credentials_test.go`

Reply:
These are local private types.  They aren't duplicates, since we can't import them.